bedrock/base/templates/includes/structured-data/organizations/mozilla-corporation-organisation.json

```diff
@@ -0,0 +1,518 @@
+  "foundingDate": "2005-08-03",
+  "founder": "Mitchell Baker",
+  "sameAs": [
+    "https://www.wikidata.org/wiki/Q169925",
+    "https://twitter.com/mozilla",
+    "https://www.facebook.com/mozilla",
+    "https://github.com/mozilla",
+    "https://www.instagram.com/mozilla/",
+    "https://www.youtube.com/user/Mozilla",
+    "https://en.wikipedia.org/wiki/Mozilla_Corporation",
+    "https://www.linkedin.com/company/mozilla-corporation"
+  ],
+  "brand": [
+  {
+    "@type": "brand",
+    "@id": "https://www.mozilla.org/#mozilla",
+    "url": "https://www.mozilla.org",
+    "logo": "{{ settings.CANONICAL_URL + static('img/mozorg/mozilla-256.jpg') }}",
+    "name": "Mozilla",
+    "sameAs": ["https://en.wikipedia.org/wiki/Mozilla", "https://www.wikidata.org/wiki/Q9661"]
+  },
+  {
+    "@type": "brand",
+    "@id": "https://www.mozilla.org/#firefoxbrand",
+    "url": "https://www.mozilla.org/firefox/",
+    "logo": "{{ settings.CANONICAL_URL + static('protocol/img/logos/firefox/logo-lg-high-res.png') }}",
+    "name": "Firefox",
+    "sameAs": [
+      "https://twitter.com/firefox",
+      "https://www.instagram.com/firefox/",
+      "https://www.youtube.com/firefox"
+    ]
+  }],
+  "owns": [
+  {
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxandroid",
+    "url": "https://www.mozilla.org/firefox/mobile/",
+    "logo": "{{ settings.CANONICAL_URL + static('img/logos/firefox/quantum/logo-lg-high-res.png') }}",
+    "image": [
+      "{{ settings.CANONICAL_URL + static('img/structured-data/browser-android-1.jpg') }}",
+      "{{ settings.CANONICAL_URL + static('img/structured-data/browser-android-2.jpg') }}",
+      "{{ settings.CANONICAL_URL + static('img/structured-data/browser-android-3.jpg') }}"
+    ],
+    "name": "Firefox Browser Android",
+    "alternateName": [
+      "Firefox Android",
+      "Firefox Browser for Android",
+      "Firefox for Android"
+    ],
+    "brand":
+    {
+      "@id": "https://www.mozilla.org/#firefoxbrand"
+    },
+    "sameAs": [
+      "https://www.wikidata.org/wiki/Q550315",
+      "https://en.wikipedia.org/wiki/Firefox_for_Android",
+      "https://play.google.com/store/apps/details?id=org.mozilla.firefox"
+    ],
+    "offers":
+    {
+      "@type": "Offer",
+      "url": "https://play.google.com/store/apps/details?id=org.mozilla.firefox",
+      "priceCurrency": "USD",
+      "price": "0",
+      "availability": "https://schema.org/InStock"
+    }
+  },
+  {
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxbeta",
+    "url": "https://www.mozilla.org/en-US/firefox/channel/desktop/",
+    "logo": "{{ settings.CANONICAL_URL + static('protocol/img/logos/firefox/browser/beta/logo-lg-high-res.png') }}",
+    "image": [
+      "https://blog.mozilla.org/futurereleases/files/2018/06/Logo.png",
+      "https://lh3.googleusercontent.com/CA7nhLfxDcKFhnItz4xdobl1LJQmwb9qLd9HjD7JdTRI0x_JwgPUCSYqhGDbhzx3MA=w3360-h1704",
```

Comment:
Like I said on slack, I would be fine with this. I dont think the code is rendered, so it would stop or delay any browser in performance, but I am fine with hosting them in bedrock.

Reply:
I don't think browser of page performance is a concern, more wherever this content might surface (in search results?)

Reply:
For search results now it would be better to use already ranking pictures and all of tehse I got from image search, but you have a point, the images might change, get deleted or altered and then we have a problem.

Reply:
Okay I'll move them all into bedrock.